devices/leviton.js

```diff
@@ -101,9 +101,9 @@ module.exports = [
                 .withDescription('Specifies the maximum brightness value'),
             exposes.numeric('ballast_power_on_level', ea.ALL).withValueMin(1).withValueMax(254)
                 .withDescription('Specifies the initialisation light level. Can not be set lower than "ballast_minimum_level"'),
-            exposes.numeric('ballast_physical_minimum_level', ea.ALL).withValueMin(1).withValueMax(254)
+            exposes.numeric('ballast_physical_minimum_level', ea.STATE_SET).withValueMin(0).withValueMax(255)
```

Comment:
@Koenkk what's the right way to make this actually read only? 

is it removing https://github.com/Koenkk/zigbee-herdsman-converters/blob/2103a26beb34e2dff1f11afc29e16d37b377db8f/converters/toZigbee.js#L581? 

I think this attribute _might_ be broadly read only, it definitely is for this model, but I haven't been able to dig up a lot of documents...

Reply:
Hm, that didn't seem to work (https://github.com/Koenkk/zigbee-herdsman-converters/pull/4813/commits/a5561a910fc382c8834565396108ad34dabb5be5)

Reply:
Read only = `ea.STATE_GET` 

- So the value can be found in the published state
- And you can do a `/get`

Reply:
Whoops, I did `STATE_SET` by accident on the first one. Updated now.

Either way, how do I get the CI to pass now - is there something else I have to change? https://github.com/Koenkk/zigbee-herdsman-converters/actions/runs/3303623754/jobs/5451785989

Thanks!

Reply:
Obsolete with direction https://github.com/Koenkk/zigbee-herdsman-converters/pull/4813#issuecomment-1288036529